### PR TITLE
Add home page in new client

### PR DESCRIPTION
**Pull request by AdityaKhatri**
## Changes

* Bring Home page to new client

## This PR doesn't introduce any:

- [x] temporary files, auto-generated files or secret keys
- [x] build works
- [x] eslint issues
- [x] typescript issues
- [x] `console.log` meant for debugging
- [x] typos
- [x] unwanted comments
- [x] conflict markers

## This PR contains valid:

- [x] permission checks
- [x] translations

### CODE REVIEW

File: app/views/Home/ProjectItem/index.tsx [hunk 1]
```diff
@@ -128,14 +113,23 @@ function ProjectItem(props: RecentProjectItemProps & PropsFromState) {
...
             date: (new Date(pa.date)).getTime(),
         })).sort((a, b) => compareDate(a.date, b.date))
     ), [projectActivity]);
 
-    const canEditProject = projectRoles[role]?.setupPermissions?.modify;
-    const canAddEntries = projectRoles[role]?.entryPermissions?.modify;
+    // const canEditProject = projectRoles[role]?.setupPermissions?.modify;
+    // const canAddEntries = projectRoles[role]?.entryPermissions?.modify;
+    const canEditProject = true;
+    const canAddEntries = true;
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]: Why set these to true?

File: app/views/Home/ProjectItem/index.tsx [hunk 2]
```diff
@@ -128,14 +113,23 @@ function ProjectItem(props: RecentProjectItemProps & PropsFromState) {
...
+
+    const routeToTagging = useRouteMatching(
+        routes.tagging,
+        {
+            projectId,
+        },
+    );
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]: Why use useRouteMatching here?

File: app/views/Home/ProjectItem/index.tsx [hunk 3]
```diff
@@ -166,7 +160,8 @@ function ProjectItem(props: RecentProjectItemProps & PropsFromState) {
                     {canEditProject && (
                         <ButtonLikeLink
                             variant="tertiary"
-                            to={reverseRoute(pathNames.editProject, { projectId })}
+                            // FIXME: Add route to new project edit later
+                            to=""
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]:
```suggestion
                            // FIXME: Add route to new project edit later
                            to="#"
```

File: app/views/Home/ProjectItem/index.tsx [hunk 4]
```diff
@@ -180,21 +175,21 @@ function ProjectItem(props: RecentProjectItemProps & PropsFromState) {
...
         >
             <div className={styles.info}>
-                <div className={styles.basicDetails}>
-                    <div className={styles.description}>
+                {(description?.length ?? 0) > 0 && (
+                    <div className={styles.basicDetails}>
                         {description}
                     </div>
-                </div>
+                )}
```
Comment (tnagorra) [inline app/views/Home/ProjectItem/index.tsx]: Why not just check for description string to be truthy?

File: app/views/Home/index.tsx [hunk 5]
```diff
@@ -1,122 +1,210 @@
...
+    startDate: projectStat.startDate,
+    endDate: projectStat.endDate,
+    projectOwnerName: projectStat.createdByName,
+    analysisFrameworkTitle: projectStat.analysisFrameworkTitle,
+    analysisFramework: projectStat.analysisFramework,
+    totalUsers: projectStat.numberOfUsers,
+    totalSources: projectStat.numberOfLeads,
+    totalSourcesTagged: projectStat.numberOfLeadsTagged,
+    totalSourcesValidated: projectStat.numberOfLeadsTaggedAndVerified,
+    projectActivity: projectStat.entriesActivity,
```
Comment (tnagorra) [inline app/views/Home/index.tsx]: This doesn't look correct

File: app/views/Home/index.tsx [hunk 6]
```diff
@@ -1,122 +1,210 @@
...
+                    <div className={styles.emptyRecentProject}>
+                        <GiShrug className={styles.icon} />
+                        <div className={styles.text}>
+                            {/* FIXME: use strings with appropriate wording */}
+                            Looks like you do not have any recent project,
+                            <br />
+                            please select a project to view it&apos;s details
+                        </div>
+                    </div>
+                )}
```
Comment (tnagorra) [inline app/views/Home/index.tsx]: Why not use List's empty icon and empty message props?

**Review by tnagorra — CHANGES_REQUESTED** [review]

File: app/views/Home/ProjectItem/index.tsx (hunk 1 above)
Comment (AdityaKhatri) [reply app/views/Home/ProjectItem/index.tsx]: No data in API right now, @k9845 is working on new queries for this.

Comment (tnagorra) [reply app/views/Home/ProjectItem/index.tsx]: Let's add a FIXME/TODO comment

**Pull request merged**